Extensions/PlatformBehavior/README.md

```diff
@@ -55,3 +55,259 @@ To detect such cases, 2 flags are used:
 For further details on the implementation, please take a look at the comments in:
 - the function `gdjs.PlatformerObjectRuntimeBehavior._findHighestFloorAndMoveOnTop`
 - the class `gdjs.PlatformerObjectRuntimeBehavior.FollowConstraintContext`
+
+## Jump trajectory
+
+### Sequential calculus of the engine
+
+The engine uses the previous speeds and positions to evaluate the new ones (with Verlet integrations). This is fast for sequential access and it's a good fit for what the engine does.
+
+The model doesn't follow the laws of physics and this is fine because more realism doesn't mean more fun. Internally, it uses 2 independent speeds:
+- the current jumping speed `currentJumpSpeed`
+- the current falling speed `currentFallSpeed`
+
+The jump speed stays at its initial value while the jump is sustained. Then, it decreases according to the gravity until it reaches 0 and stays constant until the end.
```

Comment:
nitpicking: you say decreases (which I think is fine) but the Jump Speed diagram shows that it's negative and then increase until it reaches 0.